formats/mainstream_browse_page/publisher/details.json

```diff
@@ -3,6 +3,9 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "child_ordering": {
```

Comment:
At the moment, both 1st and 2nd level pages can access the `child_ordering` parameter in `collections-publisher`. 
`child_ordering` refers to the children of the page itself, I therefore do not think that `2nd_level_ordering` is the best name because we would be ending up with 2nd level pages with a `2nd_level_ordering` param that refers to an eventual 3rd level layer.